tests/RebuilderTest.php

```diff
@@ -0,0 +1,196 @@
+<?php
+namespace Aura\Sql;
+
+
+class RebuilderTest extends \PHPUnit_Framework_TestCase
+{
+    public function testCommentedSQL()
+    {
+        $stm = "SELECT * -- Don't bind :foo
+                FROM pdotest
+                /*
+                Don't bind :bar
+                 */";
+        $values =  array(
+            'foo' => array('should', 'be', 'left', 'alone'),
+            'bar' => array('should', 'be', 'too')
+        );
+
+        $rebuilder = new Rebuilder();
+
+        list($actual, $values) = $rebuilder->rebuildStatement($stm, $values);
+
+        $expect = $stm;
+        $this->assertSame($expect, $actual);
+    }
+
+    public function testSingleLineCommentAsLastLine()
+    {
+        $stm = "SELECT *
+                FROM pdotest
+                -- Don't bind :foo";
+        $values =  array(
+            'foo' => array('should', 'be', 'left', 'alone'),
+        );
+
+        $rebuilder = new Rebuilder();
+
+        list($actual, $values) = $rebuilder->rebuildStatement($stm, $values);
+
+        $expect = $stm;
+        $this->assertSame($expect, $actual);
+    }
+
+    public function testUnclosedMultiLineComment()
+    {
+        $stm = "SELECT *
+                FROM pdotest
+                /* Don't bind :foo";
```

Comment:
It is what we want to test: that the Rebuilder does not choke on a statement with unclosed comment.